concepts/basics/.meta/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "authors": [
+    "pwadsworth"
+  ],
+  "blurb": "Learn the basic syntax of an Haskell file"
```

Comment:
so this "an" is correct if and only if Haskell is pronounced with silent H. I think we usually pronounce the H though, so can I confirm that this should be "a Haskell file" instead?

Reply:
ESL strikes again